setup.py

```diff
@@ -99,23 +99,22 @@ def get_cryptography_version():
         "pytz",
         "tzlocal",
         "aiolancium",
-    ],
+    ]
+    + REST_REQUIRES,
```

Comment:
Consider make the diff simpler (also for future changes when this gets reverted), by inserting the items instead of concatenating the containers. This restricts the diff to the actual change.
```suggestion
        *REST_REQUIRES,
    ],
```
Not that it matters here, but performance should be comparable.